src/citrine/resources/predictor.py

```diff
@@ -11,6 +13,18 @@
 CreationType = TypeVar('CreationType', bound=Predictor)
 
 
+class AutoConfigureMode(BaseEnumeration):
+    """[ALPHA] The format to use in building auto-configured assets.
+
+    * PLAIN corresponds to a single-row GEM table and plain predictor
+    * FORMULATION corresponds to a multi-row GEM table and formulations predictor
```

Comment:
Please add a line for INFER

Reply:
Added. (words are hard though, lmk if there's a more helpful tagline)